### PR TITLE
 chore: upgrade Node version needed to 14.*

### LINKED ISSUE #145: Error when running tests: this version of ember-auto-import requires the app to have a dependency on webpack
Could anyone try if you have the same problem? I believe it's related to the last release of EmberJS (3.27) when doing `npm run test`:

```
[ember-auto-import] this version of ember-auto-import requires the app to have a dependency on webpack 
```

I found it when [trying to solve this PR](https://github.com/minutebase/ember-can/pull/144#issuecomment-874186819). It seems to have a very easy solution but I need to know I'm not the only one having this issue.

### PR DESCRIPTION
**Pull request by Gorzas**
Close #145

### CODE REVIEW

**Review by Exelord — CHANGES_REQUESTED** [review]
We do not depend on webpack in anyway, so we cannot include it in dependencies.

File: package.json [hunk 1]
```diff
@@ -98,7 +98,7 @@
     "release-it-lerna-changelog": "3.1.0"
   },
   "engines": {
-    "node": "12.* || >= 14"
+    "node": ">= 14"
```
Comment (Exelord) [inline package.json:101]:
```suggestion
    "node": "14.* || >= 16"
```

We don't want to support node 15 :) only LTS

**Review by Exelord — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Exelord) [reply package.json:101]: Could you also please update readme?

Comment (Gorzas) [reply package.json]: Updated 😄

**Comment by Exelord** [comment]
There is one more thing that bothers me 🤔  why all tests are passing on node 12 on CI?

**Comment by Gorzas** [comment]
> 
> 
> There is one more thing that bothers me 🤔 why all tests are passing on node 12 on CI?

I think the problem I had was with npm v6 not with node v12. In the tests it's installing the latest NPM version.

**Comment by Gorzas** [comment]
It seems I have to do another change. Tests are failing for `ember-canary`.

**Comment by mkszepp** [comment]
@esbanarango we can close this, as we have updated to node v18

**Pull request closed without merging**